run.py

```diff
@@ -1,5 +1,7 @@
+from config import DevConfig
+
 if __name__ == '__main__':
     from app import create_app
 
     app = create_app()
-    app.run()
+    app.run(host='0.0.0.0', port=DevConfig.PORT)
```

Comment:
I expect this should be `... port=app.config['PORT'])` to prevent future changes and so we can test locally with different `APP_SETTINGS`.